pom.xml

```diff
@@ -64,6 +64,12 @@
             <version>1.19.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Waarom mockito er bij? Die wordt niet gebruikt?

Reply:
Wordt in PrometheusExporterTest gebruikt om een Realm provider te mocken.